src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Controller/EntityCreateController.php

```diff
@@ -228,7 +285,7 @@ private function getService()
      * @throws InvalidArgumentException
      */
     private function handleCopy(
-        SaveEntityCommand $command,
+        SaveSamlEntityCommand $command,
```

Comment:
Is this a TODO? The handleCopy method now only seems to support SAML entities.

Reply:
Yes, the entity copy should be added for OIDC entities to, but I wanted to do that after the entity could handle OIDC entities.